projects/foto-finder-final.md

```diff
@@ -0,0 +1,205 @@
+
+#### Desktop layout:
+
+![Foto Finder Desktop][desktop-base]
+
+
+#### Mobile layout:
+
+![Foto Finder Mobile][mobile-base]
+
+
+#### Desktop and Mobile Full View
+
+![Foto Finder Mobile Expanded][mobile-expanded]
+
+![Foto Finder Desktop Expanded][desktop-expanded]
+
+
+#### Color and font spec:
+
+![Design Specs][design-specs]
+
+[desktop-base]: /assets/images/projects/foto-finder/fotofinder-1.png
+[mobile-base]: /assets/images/projects/foto-finder/fotofinder-2.png
+[desktop-expanded]: /assets/images/projects/foto-finder/fotofinder-3.png
+[mobile-expanded]: /assets/images/projects/foto-finder/fotofinder-4.png
+[design-specs]: /assets/images/projects/foto-finder/fotofinder-5.png
+
+------------------------------------------------------------------
+
+## Rubric
+
+### Functional Expectations
+
+* [ ]  Novice - Application meets all of the basic functional expectations of create, edit, delete, and those changes persist in `localStorage`
+* [ ]  Advanced Beginner - Application adds 'Favoriting a Photo' and enables 'Filtering and Searching by Text' as defined in the spec
+* [ ]  Proficient - Applications adds 'Recent Photos' and 'View Favorites' as outlined in the spec
+* [ ]  Exceptional - Application adds at least one of the extensions
+
+------------------------------------------------------------------
+
+### Comp Recreation
+
+* [ ]  Novice - Application implements all major comp details accurately and correctly on desktop only (colors, fonts, icons, spacing, alignment, etc.)
+* [ ]  Advanced Beginner - Application implements all major comp details accurately and correctly on desktop and mobile (colors, fonts, icons,spacing, alignment,  etc.). Transitions between screen sizes may not be smooth.
+* [ ]  Proficient - Application implements all major comp details accurately and correctly on desktop and mobile (colors, fonts, icons,spacing, alignment,  etc.) with smooth transitions between screen sizes. Additional elements added generally match the visuals established in the comps, but may be slightly awkward.
+* [ ]  Exceptional - Application implements all major comp details accurately and correctly on desktop and mobile (colors, fonts, icons,spacing, alignment,  etc.) with smooth transitions between screen sizes. Additional elements that have been added match the visuals established in the comps.
+
+------------------------------------------------------------------
+
+### HTML - Style and Implementation
+
+* [ ]  Novice - Crafts markup according to the [turing html style guide](https://github.com/turingschool-examples/html)
+* [ ]  Advanced Beginner - Application adds to the above by using `data-*` attributes for all data related things
+* [ ]  Proficient - Applications adds to the above with markup that is easy to read and follow across naming conventions
+* [ ]  Exceptional - Application adds to the above by using BEM, SMACCS, or another set of naming conventions for classes
```

Comment:
Maybe look at the ideabox eval for this since it links to docs? And for css below